rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "1.71.1"
+components = [ "rustfmt", "clippy" ]
```

Comment:
Let's keep lists alphabetized pretty much anywhere it doesn't break the intended behavior.